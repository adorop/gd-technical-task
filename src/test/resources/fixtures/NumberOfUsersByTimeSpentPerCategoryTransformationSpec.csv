category,product,userId,eventTime,eventType,sessionId,sessionStartTime,sessionEndTime
books,Scala for Dummies,user 100,2018-03-01 12:00:02,view description,books_user100_1,2018-03-01 12:00:02,2018-03-01 12:06:15
books,Scala for Dummies,user 100,2018-03-01 12:01:40,like,books_user100_1,2018-03-01 12:00:02,2018-03-01 12:06:15
books,Scala for Dummies,user 100,2018-03-01 12:01:50,check status,books_user100_1,2018-03-01 12:00:02,2018-03-01 12:06:15
books,Java for Dummies,user 100,2018-03-01 12:02:14,view description,books_user100_1,2018-03-01 12:00:02,2018-03-01 12:06:15
books,Java for Dummies,user 100,2018-03-01 12:02:45,dislike,books_user100_1,2018-03-01 12:00:02,2018-03-01 12:06:15
books,Java for Dummies,user 100,2018-03-01 12:02:50,close description,books_user100_1,2018-03-01 12:00:02,2018-03-01 12:06:15
books,Scala for Dummies,user 100,2018-03-01 12:04:30,view description,books_user100_1,2018-03-01 12:00:02,2018-03-01 12:06:15
books,Scala for Dummies,user 100,2018-03-01 12:06:15,add to bucket,books_user100_1,2018-03-01 12:00:02,2018-03-01 12:06:15
mobile phones,iPhone 8 Plus,user 100,2018-03-01 12:06:49,view description,mobilephones_user100_1,2018-03-01 12:06:49,2018-03-01 12:08:01
mobile phones,iPhone 8 Plus,user 100,2018-03-01 12:07:14,add to bucket,mobilephones_user100_1,2018-03-01 12:06:49,2018-03-01 12:08:01
mobile phones,iPhone 8,user 100,2018-03-01 12:08:01,view description,mobilephones_user100_1,2018-03-01 12:06:49,2018-03-01 12:08:01
mobile phones,iPhone 8,user 100,2018-03-01 12:13:10,dislike,mobilephones_user100_2,2018-03-01 12:13:10,2018-03-01 12:13:10
notebooks,MacBook Pro 15,user 100,2018-03-01 12:15:08,view description,notebooks_user100_1,2018-03-01 12:15:08,2018-03-01 12:16:10
notebooks,MacBook Pro 15,user 100,2018-03-01 12:16:10,check status,notebooks_user100_1,2018-03-01 12:15:08,2018-03-01 12:16:10
books,"Sherlock Holmes, full collection",user 200,2018-03-01 12:11:25,view description,books_user200_1,2018-03-01 12:11:25,2018-03-01 12:15:10
books,"Sherlock Holmes, full collection",user 200,2018-03-01 12:12:15,check status,books_user200_1,2018-03-01 12:11:25,2018-03-01 12:15:10
books,"Sherlock Holmes, full collection",user 200,2018-03-01 12:13:15,sign for updates,books_user200_1,2018-03-01 12:11:25,2018-03-01 12:15:10
books,Romeo and Juliet,user 200,2018-03-01 12:15:10,view description,books_user200_1,2018-03-01 12:11:25,2018-03-01 12:15:10
books,Romeo and Juliet,user 200,2018-03-01 12:15:10,check status,books_user200_1,2018-03-01 12:11:25,2018-03-01 12:15:10
books,Romeo and Juliet,user 200,2018-03-01 12:15:10,add to bucket,books_user200_1,2018-03-01 12:11:25,2018-03-01 12:15:10
notebooks,MacBook Pro 13,user 200,2018-03-01 12:15:33,view description,notebooks_user200_1,2018-03-01 12:15:33,2018-03-01 12:17:56
notebooks,MacBook Pro 13,user 200,2018-03-01 12:17:25,like,notebooks_user200_1,2018-03-01 12:15:33,2018-03-01 12:17:56
notebooks,MacBook Pro 13,user 200,2018-03-01 12:17:56,add to bucket,notebooks_user200_1,2018-03-01 12:15:33,2018-03-01 12:17:56
mobile phones,iPhone X,user 300,2018-03-01 12:05:03,view description,mobilephones_user300_1,2018-03-01 12:05:03,2018-03-01 12:05:20
mobile phones,iPhone X,user 300,2018-03-01 12:05:20,add to bucket,mobilephones_user300_1,2018-03-01 12:05:03,2018-03-01 12:05:20
notebooks,MacBook Air,user 300,2018-03-01 12:14:58,view description,notebooks_user300_1,2018-03-01 12:14:58,2018-03-01 12:14:58
notebooks,MacBook Air,user 300,2018-03-01 12:20:10,dislike,notebooks_user300_2,2018-03-01 12:20:10,2018-03-01 12:20:10
