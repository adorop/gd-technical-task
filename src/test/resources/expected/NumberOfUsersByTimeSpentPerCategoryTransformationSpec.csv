category,lt_one,one_to_five,gt_five
books,0,1,1
mobile phones,2,1,0
notebooks,1,2,0